tests/Resources/Page/OCPDashboard/ConfigMaps/ConfigMaps.robot

```diff
@@ -20,17 +20,17 @@ Check If ConfigMap Exists
     [Return]   ${status}
 
 Get PVC Size
-    [Documentation]    Get configure PVC size from configmap
-    [Arguments]   ${namespace}   ${configmap_name}=jupyterhub-cfg
-    ${data}    OpenShiftCLI.Get  kind=ConfigMap  namespace=${namespace}
+    [Documentation]    Get configure PVC size from OdhDashboardConfig CR
+    [Arguments]   ${namespace}   ${configmap_name}=odh-dashboard-config
+    ${data}    OpenShiftCLI.Get  kind=OdhDashboardConfig  namespace=${namespace}
```

Comment:
Could you use OpenShiftLibrary here?

Reply:
Jorge i would rather change all the things together. There are other keyword which i am not using i am not sure where those keyword has been used. Rather changing for this keyword i would do while we do refactoring .

Reply:
I think it'd be safe enough to replace now this call